### ISSUE TITLE
[Reminder]: Some hooks deprecated since Rollup v1.0.0

### ISSUE THREAD

**Issue by DrSensor** [ezolenko/rollup-plugin-typescript2#126]
#### What happens and why it is wrong
https://github.com/rollup/rollup/releases/tag/v1.0.0
```js
(!) rpt2 plugin: The ongenerate hook used by plugin rpt2 is deprecated. The generateBundle hook should be used instead.
(!) rpt2 plugin: The onwrite hook used by plugin rpt2 is deprecated. The generateBundle hook should be used instead.
```

#### Versions
- typescript: 3.2.2
- rollup: 1.0.0
- rollup-plugin-typescript2: 0.18.1

**Comment by ufocoder** [comment]
Hooks used by `rollup-plugin-typescript2` [have been deprecated](https://github.com/rollup/rollup/blob/master/docs/05-plugins.md#deprecated)

**Comment by JoshLipan** [comment]
Same warning

**Comment by aladdin-add** [comment]
seem it has been fixed on master, mind releasing a patch? @ezolenko

**Comment by ezolenko** [comment]
Yep, this is fixed in 19.0+, I'll do a release soonish

**Comment by ezolenko** [comment]
In 0.19.0 now
